packages/shared/styles/color.ts

```diff
@@ -100,6 +100,50 @@ export const pickReadableTextColor = (
     : invertedTextOption;
 };
 
+/*
+ * This function will select a WCAG-compliant text color based on the background color.
+ * Reference: https://wunnle.com/dynamic-text-color-based-on-background
+ * TODO: We should deprecate pickReadableTextColor and update components that use it.
```

Comment:
I would like to separate this out because it is not quite related to this work. I want to test out all of the components it's used with thoroughly and make sure that's okay with the design team. For example, our SuccessButton will now have dark text instead of white if I update this. I'm not sure what the rest of the impact will be on other components and want to be able to document that. It will also be better for reviewing.